docs/source/admin/traffic_ops/configuration.rst

```diff
@@ -245,7 +245,7 @@ Below is a list of cache parameters for special configuration, which are unlikel
 +==========================+===================+=========================================================================================================================+
 | not_a_parent             | parent.config     | This is a boolean flag and is considered 'true' if it exists and has any value except 'false'.                          |
 |                          |                   | This prevents servers with this parameter in their profile from being inserted into the ``parent.config`` generated for |
-|                          |                   | servers with this server's Cache Group as a parent of their CacheGroup. This is primarily useful for when edge caches   |
+|                          |                   | servers with this server's Cache Group as a parent of their Cache Group. This is primarily useful for when edge caches   |
```

Comment:
This table is malformed. It should also link "Cache Group" to the glossary. (It should also use the `.. table` directive but that's not your problem)

Reply:
done